storybookjs/storybook
# #2128 Add platform option for haul bundler

Issue:
#2128

## What I did
Add platform option to storybook-cli;
## How to test
storybook start -p 7007 --haul webpack.haul.storybook.js
storybook start -p 7007 --haul webpack.haul.storybook.js --platform ios
storybook start -p 7007 --haul webpack.haul.storybook.js --platform android

Is this testable with jest or storyshots?
No
Does this need a new example in the kitchen sink apps?
No
Does this need an update to the documentation?
Yes
If your answer is yes to any of these, please make sure to include it in your PR.
